### PR TITLE
[BE-3152]: Added FOD docs

### PR DESCRIPTION
**Pull request by boztopuz**

### CODE REVIEW

**Comment by boztopuz** [comment]
@ftmzhrsckldr could you please check the Android side requirements that I mentioned in documentation.

File: docs/workflows/common-workflow-steps/fod-mobile-assesment.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+---
+
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Fortify on Demand Mobile Assessment
+
+[Fortify on Demand Mobile Assessment](https://www.opentext.com/products/fortify-on-demand) provides a robust solution for securing mobile applications by identifying vulnerabilities before they can be exploited. This comprehensive cloud-based service streamlines the security testing process, ensuring your mobile apps meet the highest standards of security.
+
+Fortify on Demand Mobile Assessment can be seamlessly integrated into your workflow with Appcircle, enabling easy setup and utilization within your existing development processes.
```
Comment (erenkan) [inline docs/workflows/common-workflow-steps/fod-mobile-assesment.md:15]:
```suggestion
You can seamlessly integrate Fortify on Demand Mobile Assessment into your workflow with Appcircle, facilitating easy setup and utilization within your existing development processes.
```

File: docs/workflows/common-workflow-steps/fod-mobile-assesment.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+### Prerequisites
+
+Before running the **Fortify on Demand Mobile Assessment** step, certain prerequisites must be completed. These prerequisites, detailed in the table below:
```
Comment (erenkan) [inline docs/workflows/common-workflow-steps/fod-mobile-assesment.md:19]:
```suggestion
Before running the **Fortify on Demand Mobile Assessment** step, you must complete certain prerequisites, as detailed in the table below:
```

File: docs/workflows/common-workflow-steps/fod-mobile-assesment.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+| Prerequisite Workflow Step                      | Description                                     |
+|-------------------------------------------------|-------------------------------------------------|
+| [**Xcodebuild for Devices**](/workflows/ios-specific-workflow-steps/xcodebuild-for-devices) | This step will build your application in ARM architecture and generate an `IPA` and `dSYM` file. |
+| [**Android Build**](/workflows/android-specific-workflow-steps/android-build) | The app required for this step is generated by the **Android Build** (or alternative build steps). |
+| [**Android Sign**](/workflows/android-specific-workflow-steps/android-sign) | If a signed app is created, this step needs to be run beforehand to process this output.|
```
Comment (erenkan) [inline docs/workflows/common-workflow-steps/fod-mobile-assesment.md:25]:
```suggestion
| [**Android Sign**](/workflows/android-specific-workflow-steps/android-sign) | If you create a signed app, you must run this step beforehand to process the output. |
```

File: docs/workflows/common-workflow-steps/fod-mobile-assesment.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+We recommend using [**Environment Variables**](/environment-variables/managing-variables) groups for such sensitive variables.
+
+:::
+
+| Variable Name                 | Description                                                                                               | Status   |
+|-------------------------------|-----------------------------------------------------------------------------------------------------------|----------|
+| `$AC_FOD_CLIENT_ID`           | Authenticating with Client Credentials. The client credentials are the API keys generated in the portal   | Required |
+| `$AC_FOD_CLIENT_SECRET`       | Authenticating with Client Credentials. The client credentials are the API keys generated in the portal.  | Required |
+| `$AC_FOD_CLIENT_DATACENTER`   | Select the datacenter assigned to you. This is provided once your are onboarded with Fortify on Demand.   | Required |
+| `$AC_FOD_ENTITLEMENT_ID`      | The ID assigned to ypur current entitlement. This value can be reterived in Foritfy on Demand portal.     | Required |
```
Comment (erenkan) [inline docs/workflows/common-workflow-steps/fod-mobile-assesment.md:54]:
```suggestion
| `$AC_FOD_ENTITLEMENT_ID`      | The ID assigned to your current entitlement. You can retrieve this value in the Fortify on Demand portal.     | Required |
```

File: docs/workflows/common-workflow-steps/fod-mobile-assesment.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+We recommend using [**Environment Variables**](/environment-variables/managing-variables) groups for such sensitive variables.
+
+:::
+
+| Variable Name                 | Description                                                                                               | Status   |
+|-------------------------------|-----------------------------------------------------------------------------------------------------------|----------|
+| `$AC_FOD_CLIENT_ID`           | Authenticating with Client Credentials. The client credentials are the API keys generated in the portal   | Required |
+| `$AC_FOD_CLIENT_SECRET`       | Authenticating with Client Credentials. The client credentials are the API keys generated in the portal.  | Required |
+| `$AC_FOD_CLIENT_DATACENTER`   | Select the datacenter assigned to you. This is provided once your are onboarded with Fortify on Demand.   | Required |
```
Comment (erenkan) [inline docs/workflows/common-workflow-steps/fod-mobile-assesment.md:53]:
```suggestion
| `$AC_FOD_CLIENT_DATACENTER`   | Select the datacenter assigned to you upon your onboarding with Fortify on Demand.   | Required |
```

**Review by erenkan — CHANGES_REQUESTED** [review]

**Review by erenkan — APPROVED** [review]

File: docs/workflows/common-workflow-steps/fod-mobile-assesment.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Fortify on Demand Mobile Assessment
+
+[Fortify on Demand Mobile Assessment](https://www.opentext.com/products/fortify-on-demand) provides a robust solution for securing mobile applications by identifying vulnerabilities before they can be exploited. This comprehensive cloud-based service streamlines the security testing process, ensuring your mobile apps meet the highest standards of security.
+
+You can seamlessly integrate Fortify on Demand Mobile Assessment into your workflow with Appcircle, facilitating easy setup and utilization within your existing development processes.
+
+### Prerequisites
```
Comment (ftmzhrsckldr) [inline docs/workflows/common-workflow-steps/fod-mobile-assesment.md:17]: Flutter iOS and Android should be added too. 
The Prerequisites table should be under each platform heading.
You can examine it as an example:
- https://docs.appcircle.io/workflows/common-workflow-steps/firebase-app-distribution#prerequisites

**Pull request merged**